inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/scheduling/TaskConsumer.java

```diff
@@ -66,7 +67,7 @@ public void run()
                     activeTask.run();
                     log.debug("Indexing task completed: {}", activeTask);
                 }
-                catch (Throwable e) {
+                catch (BeansException e) {
```

Comment:
Same comment as in `Selectiontask`

Reply:
Catching Throwable is intentional here as we want to continue the execution even if a particular recommender fails.